CityOfDetroit/bloom
# fix: show all listings button labels

# Pull Request Template

## Issue Overview

This PR addresses #1543 


## Description
Changed label of reset button to 'reset'. Added aria-label to filter button.
Removed obsolete translations from used keys.

## How Can This Be Tested/Reviewed?

go to listings page, turn on screen reader. Check filter button without filters + after picking some filters, and reset button.

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
